src/sonic-config-engine/minigraph.py

```diff
@@ -1717,6 +1718,19 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
             results['LOOPBACK_INTERFACE'][host_lo_intf[0]] = {}
 
     results['MGMT_VRF_CONFIG'] = mvrf
+    # Update SNMP_AGENT_ADDRESS_CONFIG with Management IP and Loopback IP
+    # if available.
+    if not is_multi_asic() and asic_name is None:
```

Comment:
>if [](http://example.com/codeflow?start=4&length=2)

If the condition does not meet, please initialize an empty `results['SNMP_AGENT_ADDRESS_CONFIG']`.